content/07.pathogenesis.md

```diff
@@ -446,17 +445,29 @@ Data and empirically determined biological mechanisms inform models, while model
 The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling future outbreaks and tracing the path of the spread of SARS-CoV-2.
-Efforts vary widely by country: the UK has coordinated has coordinated a national database of viral genomes [@doi:10/d5mg]; no such coordination has been achieved in the USA.
+Efforts vary widely by country: the UK has coordinated a national database of viral genomes [@doi:10/d5mg]; no such coordination has been achieved in the USA.
 Several studies used phylogenetic analysis to determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New York City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
 There is an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of January 18, 2021 more than 381,000 genome sequences have been collected from patients.<!-- TODO: consider auto-updating this -->
 The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2/], and COVID-19 data portal [@https://www.covid19dataportal.org/].
 
+While the evolution of SARS-CoV-2 variants are discussed elsewhere within this special issue [], some of these variants may be associated with increased transmission of SARS-CoV-2. 
+For example, the coordinated sequencing efforts in the U.K. were instrumental in detecting B.1.1.7/VOC 202012/01, a variant of concern (VOC) that is associated with approximately 50% increased transmission with an estimated R0 of 1.4 due to alterations of the spike protein as a result of genetic mutations [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/955239/NERVTAG_paper_on_variant_of_concern__VOC__B.1.1.7.pdf; @doi:10.1101/2020.12.24.20248822; @doi:10.1016/j.cell.2021.01.044].
+Since spreading to at least 93 other countries [@url:https://cov-lineages.org/global_report_B.1.1.7.html], other studies investigating its transmission have occured.
+In the U.S. it was estimated that B.1.1.7 had an increased transmission of 35-45% versus the common SARS-CoV-2 variants at the time between December 2020 and January 2021, and B.1.1.7 is on its way to become the dominant SARS-CoV-2 variant in the U.S. [@doi:10.1101/2021.02.06.21251159].
+Notably, the latest analyses in the U.K. reports that there may be an increased risk of disease severity associated with B.1.1.7 [@url:https://www.gov.uk/government/publications/nervtag-paper-on-covid-19-variant-of-concern-b117] and potentially a 35% increased risk of death [@doi:10.1101/2021.02.01.21250959] versus common variants of SARS-CoV-2. 
+However, further studies are required to confirm these associations. 
+Other variants have been identified including the P.1 lineage associated with outbreaks in Brazil and B.1.351 a variant first identified in South Africa.
```

Comment:
```suggestion
Other variants meriting further investigation have also been identified worldwide, including the P.1 lineage associated with outbreaks in Brazil and the B.1.351 variant first identified in South Africa.
```
Not sure if my phrasing of not-VOC-but-we're-worried makes sense!

Reply:
Haha it does, at this stage they are all colloquially VOCs